decidim-proposals/app/views/decidim/proposals/proposal_votes/update_buttons_and_counters.js.erb

```diff
@@ -17,18 +17,18 @@
 <% if vote_limit_enabled? %>
   (function() {
     var $remainingVotesCount = $('#remaining-votes-count');
-    var $notVotedButtons = $('.card__button.button').not('.success');
+    var $notVotedButtons = $('.vote-btn').not('[id^=vote_button-]');
```

Comment:
As `vote-btn` is a selector for Javascript, I think this should be a data attribute, so we can differentiate the selectors for JS and CSS. 

Maybe something like `data-proposal-vote-button` ? Feel free to improve it

Reply:
Also let's migrate to data attributes the `#remaining-votes-count` selector too